src/Arcus.Scripting.LogicApps/Arcus.Scripting.LogicApps.psm1

```diff
@@ -0,0 +1,23 @@
+<#
+ .Synopsis
+  Disable all specified Logic Apps described in the order control JSON file.
+  
+ .Description
+  Disables all specified Logic Apps in a specific order. The Logic Apps to be disabled and the order in which this will be done, will be defined in the deploy-OrderControl.json-file.
```

Comment:
```suggestion
  Disables all specified Logic Apps in a specific order. The Logic Apps to be disabled and the order in which this will be done, will be defined in the configuration file (e.g. deploy-orderControl.json).
```